x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/annotations/AnnotationPersister.java

```diff
@@ -74,7 +74,7 @@ public Builder bulkPersisterBuilder(String jobId) {
     public class Builder {
 
         private final String jobId;
-        private BulkRequest bulkRequest = new BulkRequest(AnnotationIndex.WRITE_ALIAS_NAME);
+        private BulkRequest bulkRequest = new BulkRequest(AnnotationIndex.WRITE_ALIAS_NAME).requireAlias(true);
```

Comment:
Just to make sure: would you also like to add the `requireAlias(true)` to the index requests in integration tests (like `RevertModelSnapshotIT.java` in line 217)? Or do you find it unnecessary?

Reply:
I did not really think about adding it to integration tests. I can look in there and see whats up.